inmaputil/download_test.go

```diff
@@ -0,0 +1,43 @@
+package inmaputil
+
+import "testing"
+
+func TestDownloadLocal (t *testing.T) {
+	if k := Download("/dev/null"); k != "/dev/null" {
+		t.Error("Expected /dev/null, got ", k);
+	}
+}
+
+func TestDownloadLocal2 (t *testing.T) {
+	// dummy directory
+	if k := Download("/blah/test/"); k != "/blah/test/" {
+		t.Error("Expected /blah/test/, got ", k);
+	}
+
+	t.Logf("Passed directory tests. ");
```

Comment:
This is probably not necessary because the testing framework will tell you it passes if it doesn't fail.